playbook.yml

```diff
@@ -88,8 +88,6 @@
         - '$'
         - gallery-api/
         - gallery_settings/
-        - cell/
```

Comment:
These aren't directly related, but due to the multiple "sub-OMERO's" that are presented on IDR. :+1: